### ISSUE TITLE
[low] Chrome + Talkback: Screen reader announce 'Text field label' twice. 

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8614]
## Action Performed:
1. Using Android Talkback + Chrome browser, open the URL staging.new.expensify.com/r/85021985   
2. Navigate to the settings using Swipe Gesture.          
3. Go to the payment option and Double tap on it once you reach the debit card. 
4. Navigate the text field using swipe gesture and listen to the screen reader announcement.

## Expected Result:
Screen reader should announce Text field label only once.

## Actual Result:
Screen reader announce 'Text field label' twice. 

**Other occurrences**
Same issue repro on PR  #8758, #8826, #8868, #8802, #9207

## Workaround:
Yes

## Area issue was found in:
Payments

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users hear the announcements twice that is annoying and confusing.

## Suggested resolution:
Ensure the 'Text field label' announced only once by the screen reader.            
Refer to: 
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

## Platform:
- Mobile Web

**Version Number:**  v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/163104368-f139242e-9113-4253-9a05-501714a724d5.mp4
